test/integration/swift/helm.yaml

```diff
@@ -10,10 +10,6 @@ deployVariables:
     value: "testapp"
   - name: "IMAGENAME"
```

Comment:
we shoudl rework the integration test generation to no longer be a big bash script and maybe leverage go natively